md.obsidian.Obsidian.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>md.obsidian.Obsidian</id>
+  <launchable type="desktop-id">md.obsidian.Obsidian.desktop</launchable>
+  <name>Obsidian</name>
+  <summary>Markdown-based knowledge base</summary>
+  <description>
+    <p>Obsidian is a powerful knowledge base that works on top of a local folder of plain text Markdown files.</p>
+    <p>This distribution is currently in beta and not officially supported by the Obsidian team</p>
+    <p>Note: Regular users should disable "Automatic Updates" within Obsidian as public release updates are handled by Flatpak itself. Insiders should keep this enabled to receive insider builds, as those will not be made available on Flathub.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://obsidian.md/images/screenshot.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="0.9.2" date="2020-09-30"/>
+  </releases>
+  <url type="homepage">https://obsidian.md/</url>
+  <url type="help">https://obsidian.md/community</url>
+  <url type="faq">https://obsidian.md/faq</url>
+  <url type="bugtracker">https://github.com/flathub/md.obsidian.Obsidian/issues</url>
+  <developer_name>Obsidian</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
If it's proprietary, the manifest should use `extra-data` to download it during installation, unless we are granted redistribution rights.

Reply:
Obsidian maker here. Redistribution is fine as long as we can make it clear that the same EULA (https://obsidian.md/eula) applies to the end user. Anything we need to do on our end for this?

Reply:
Thanks. Done via https://github.com/flathub/flathub/pull/1883/commits/0e61bc67faadd8a79e26aac452f988e352462608.